packages/client/src/StreamIDBuilder.ts

```diff
@@ -49,6 +49,7 @@ function parseRawDefinition(definition: StreamDefinition): [string, number | und
  * */
 export const formStreamDefinitionDescription = (definition: StreamDefinition): string => {
     const raw = parseRawDefinition(definition)
+    // eslint-disable-next-line prefer-destructuring
```

Comment:
we should just disable this rule... whether destructuring or classic variable assignment is more readable is a context-dependent choice imo

Reply:
👍